### PR TITLE
Add lang support

### PR DESCRIPTION
**Pull request by JoshuaKlassen**
Add basic lang support

US: https://rally1.rallydev.com/#/95185052972/detail/userstory/105234122516

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
+    "test",
+    "tests"
+  ],
+  "dependencies": {
+    "d2l-icons": "^3.0.0",
+    "d2l-colors": "^2.2.3",
+    "d2l-dropdown": "^5.0.5",
+    "d2l-menu": "^0.3.0",
+    "app-localize-behavior": "^1.0.0"
+  }
```
Comment (JoshuaKlassen) [inline bower.json]: adding app-localze-behavior dependency should have been the only change, not sure why everything is changed, editorconfig weirdness?

File: d2l-user-switcher.html [hunk 2]
```diff
@@ -1,91 +1,92 @@
...
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../d2l-dropdown/d2l-dropdown.html">
+<link rel="import" href="../d2l-dropdown/d2l-dropdown-menu.html">
+<link rel="import" href="../d2l-icons/d2l-icons.html">
+<link rel="import" href="../d2l-menu/d2l-menu.html">
+<link rel="import" href="../d2l-menu/d2l-menu-item.html">
+<link rel="import" href="../d2l-menu/d2l-menu-item-link.html">
+<link rel="import" href="./d2l-user-switcher-item.html">
+<link rel="import" href='./localize-behavior.html'>
+
```
Comment (JoshuaKlassen) [inline d2l-user-switcher.html]: Same thing here

**Comment by trevorjonesd2l** [comment]
You can view it [here](https://github.com/Brightspace/user-switcher/pull/4/files?w=1) for a cleaner diff with no whitespace conflicts

**Comment by trevorjonesd2l** [comment]
I believe the strings that need lang terms are here:

https://github.com/Brightspace/user-switcher/blob/master/d2l-user-switcher.html#L59

> Switch user: {{User Name}} selected

https://github.com/Brightspace/user-switcher/blob/master/d2l-user-switcher.html#L65

> User Switcher menu

**Comment by trevorjonesd2l** [comment]
I think the general trend has been to use the English term as a default until the other lang terms are updated

**Pull request merged**